tools/jslint/check-all.js

```diff
@@ -60,7 +60,7 @@ var globals =
     + ' viewport process_loaded_for_unread'
     + ' have_scrolled_away_from_top'
     + ' maybe_scroll_to_selected recenter_pointer_on_display suppress_scroll_pointer_update'
-    + ' fast_forward_pointer recent_subjects unread_subjects'
+    + ' fast_forward_pointer recent_subjects unread_subjects recent_private_messages'
```

Comment:
I've run into a problem with changing from a global variable to message_store.recent_private_messages: the node_tests/stream_list.js file was defining its own set_global('recent_public_messages', new global.Array()); so now recent_public_messages is not defined for that file.

I'm not sure what the right fix for that is - I have tried:
* seeing if the test framework would let me define a global named 'message_store.recent_private_messages'. Nope
* require('message_store.js') as well as require('stream_list.js') in the test file. Brings in new dependencies (util) but adding that doesn't seem to work. 
```
/srv/zulip/static/js/message_store.js:573
util.execute_early(function () {
^
ReferenceError: util is not defined
```

Have there been any new variables added recently that would have been globals except for the new policy? Can you point me towards a sample test using one?  Thanks